src/BinaryParsers/ElfBinary/Dwarf/DwarfCommonInformationEntry.cs

```diff
@@ -173,9 +173,9 @@
                     AddressSize = defaultAddressSize;
                     SegmentSelectorSize = 0;
                 }
-                CodeAlignmentFactor = data.LEB128();
-                DataAlignmentFactor = data.LEB128();
-                ReturnAddressRegister = data.LEB128();
+                CodeAlignmentFactor = data.ULEB128();
```

Comment:
>ULEB128 [](http://example.com/codeflow?start=43&length=7)

I changed this name as the DWARF spec itself is constantly referring to ULEB so I found this clearer.